### PR TITLE
Prefer method name as helper name, if defined

### PR DESCRIPTION
**Pull request by nicolaisueper**
This PR will implement the behavior discussed in #9.

I also updated the dependencies `extend` and `standard` to the latest version using `npm-check-updates`.

Please let me know if you're unhappy with something.

I look forward to install the next version of this plugin. 😁 

Greets
Nico

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,13 +22,13 @@ function plugin (options) {
...
 
         switch (typeof helperContents) {
-        case 'function':
-          templateName = file.split('.').shift()
-          Handlebars.registerHelper(templateName, helperContents)
-          break
-        case 'object':
-          Handlebars.registerHelper(helperContents)
-          break
+          case 'function':
```
Comment (mischah) [inline index.js]: You have forced whitespace changes 😗

Comment (nicolaisueper) [reply index.js]: Oh, thanks!

Comment (nicolaisueper) [reply index.js]: @mischah i checked it (https://github.com/nicolaisueper/metalsmith-register-helpers/blob/aff435942104dc443fdc6a68741d6eb5402d22ec/index.js) and i changed it according to the 'standard' code style applied on this project. This should be right.

Comment (nicolaisueper) [reply index.js]: @kennethormandy please check this :)

Comment (mischah) [reply index.js]: ah. Sorry. 👌

**Comment by kennethormandy** [comment]
Thanks for the Pull Request! This is published now:

```sh
npm install -D metalsmith-register-helpers@0.4.0
```

**Comment by nicolaisueper** [comment]
You're welcome. :)

**Pull request merged**